spec/models/item_spec.rb

```diff
@@ -3,5 +3,28 @@
 RSpec.describe Item, type: :model do
   describe 'relationships' do
     it { should belong_to :supermarket }
+
+  end
+
+  it "can have many customers" do
+    supermarket = Supermarket.create!(name: "ABC Market", location: "123 Sesame Street")
+    corey = Customer.create!(name: "Corey")
+    elmer = Customer.create!(name: "Elmer")
+    cereal = supermarket.items.create!(name: "Cereal", price: 3, customers: [corey, elmer])
+    prime_rib = supermarket.items.create!(name: "Prime Rib", price: 21, customers: [corey, elmer])
+
+    expect(cereal.customers).to include(corey, elmer)
+  end
+
+  describe "#customer_count" do
```

Comment:
Great work testing the instance method here too! We need to test it at the unit level (here)  and at the integration level (feature test). Nice work!